### PR TITLE
Make suffix configurable. Fixes #2

### LINKED ISSUE #2: make the -nodejs suffix configureable
I need a tool just like this for use on an electron project, but the `-nodejs` suffix will get in the way.

I could just `replace()` the output, but an option to specify the suffix in this lib would be nice.

### PR DESCRIPTION
**Pull request by simonkberg**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -44,13 +44,15 @@ const linux = name => {
 	};
 };
 
-module.exports = name => {
+module.exports = (name, suffix) => {
```
Comment (sindresorhus) [inline index.js]: Can you make suffix and option in an options object instead. To make it easier to add other options in the future.

File: readme.md [hunk 2]
```diff
@@ -22,18 +22,42 @@ paths.data;
...
 
 #### name
 
 Type: `string`
 
 Name of your project. Used to generate the paths.
 
+#### suffix = 'nodejs'
+
+Type: `mixed`
```
Comment (sindresorhus) [inline readme.md]: String

User can just pass an empty string to not have it.

File: readme.md [hunk 3]
```diff
@@ -22,18 +22,42 @@ paths.data;
...
 Type: `string`
 
 Name of your project. Used to generate the paths.
 
+#### suffix = 'nodejs'
+
+Type: `mixed`  
+
+Optional suffix to append to project name. Can be a custom string or a falsy
+value to disable. Default is 'nodejs'.
```
Comment (sindresorhus) [inline readme.md]: Should explain why suffix is needed. See explanation in the original issue.

File: index.js [hunk 4]
```diff
@@ -44,13 +44,15 @@ const linux = name => {
...
 	if (typeof name !== 'string') {
 		throw new TypeError(`Expected string, got ${typeof name}`);
 	}
 
-	// add suffix to prevent possible conflict with native apps
-	name += '-nodejs';
+	if (typeof suffix === 'undefined' || typeof suffix === 'string') {
```
Comment (sindresorhus) [inline index.js]: This check is moot.

File: readme.md (hunk 2 above)
Comment (simonkberg) [inline readme.md]: ugh, good point

**Comment by simonkberg** [comment]
@sindresorhus do you think the hyphen should be added when there's a suffix or should it be part of the suffix value? re: moot

**Comment by sindresorhus** [comment]
Hyphen should be added for the user when suffix is specified.

**Comment by simonkberg** [comment]
Alright, ready for another review!

**Comment by sindresorhus** [comment]
Thank you :)

**Comment by zeke** [comment]
Thanks, @simonkberg! I was going to do this work myself, but I'm glad I waited a couple days. :)

**Pull request merged**